profile_inference.py

```
@@ -695,6 +713,605 @@ def run_benchmark_mode(dit_handler, llm_handler, args, timer: PreciseTimer):
+            # Skip if the tier already doesn't use quantization (no point re-testing)
+            if gpu_config.quantization_default:
+                print(f"\n  --- Variant: no-quant (offload={gpu_config.offload_to_cpu_default}) ---")
+                result_no_quant = _run_single_tier_test(
+                    sim_gb, gpu_config, args, example_data,
+                    checkpoint_dir, disk_lm_models,
+                    quantization_override=None,
+                    test_variant="no-quant",
+                )
+                all_results.append(result_no_quant)
+            else:
+                print(f"\n  --- Variant: no-quant — SKIPPED (tier already has quantization=False) ---")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)